README.md

```
@@ -1,10 +1,11 @@
 HCO Phisher 🎣
 
-A powerful phishing tool with 35+ templates, created for educational purposes and security awareness.
+A powerful phishing tool with 35+ templates, created for educational purposes and security awareness.</br>
+</br>
+<img src="https://img.shields.io/badge/HCO-Phisher-red"></br>
```

markdownlint-cli2 (0.22.0)

[warning] Images should have alternate text (alt text)

(MD045, no-alt-text)